src/tree/ConcatenatedModule.js

```diff
@@ -14,6 +14,22 @@ export default class ConcatenatedModule extends Module {
     this.fillContentModules();
   }
 
+  get parsedSize() {
+    return this.getParsedSize() ?? this.getEstimatedSize('parsedSize');
+  }
+
+  get gzipSize() {
+    return this.getGzipSize() ?? this.getEstimatedSize('gzipSize');
+  }
+
+  getEstimatedSize(sizeType) {
+    const parentModuleSize = this.parent[sizeType];
+
+    if (parentModuleSize !== undefined) {
+      return Math.floor((this.size / this.parent.size) * parentModuleSize);
+    }
+  }
```

Comment:
This is taken from here: https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/a4c304e6c2d1c59a57dde6e747f9ce8e4b46a077/src/tree/ContentModule.js#L18-L24